Sprint-3/implement/test/is-valid-traingle.test.js

```diff
@@ -0,0 +1,16 @@
+const isValidTriangle = require("../is-valid-triangle");
+
+test("Is a valid triangle", () => {
+  expect(isValidTriangle(3, 3, 3)).toBe(true);
+});
+
+test("Invalid Triangle with Negative", () => {
```

Comment:
This test is correct. However, your code would not pass this test because it doesn't include a check for negative side lengths.

Reply:
return a > 0 && b > 0 && c > 0 && a + b > c && b + c > a && a + c > b;
is it good right ? for negative check